src/ErrorProne.NET.StructAnalyzers.Tests/MakeStructMemberReadOnlyCodeFixProviderTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿using ErrorProne.NET.TestHelpers;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using VerifyCS = ErrorProne.NET.TestHelpers.CSharpCodeFixVerifier<
+    ErrorProne.NET.StructAnalyzers.MakeStructMemberReadOnlyAnalyzer,
+    ErrorProne.NET.StructAnalyzers.MakeStructMemberReadOnlyCodeFixProvider>;
+
+namespace ErrorProne.NET.StructAnalyzers.Tests
+{
+    [TestFixture]
+    public class MakeStructMemberReadOnlyCodeFixProviderTests
```

Comment:
💡 Code fix tests shouldn't be separated from analyzer tests. Separating them means you're either losing test coverage or duplicating work.

Reply:
Even though I see the point of keeping these tests together, I actually think it makes sense to keep them separate.

My logic is: I tend to cover the corner cases for the analyzers in the analyzer's tests and have a few tests for the fixer to cover only the cases related to code transformation.

I tend to have significantly more code in the analyzers and the fixers usually significantly simpler.